common-java-libs/org.wso2.apk.apimgt.impl/src/main/java/org/wso2/apk/apimgt/impl/dao/impl/ApplicationDAOImpl.java

```diff
@@ -140,6 +141,19 @@ private String getGroupId(Connection connection, int applicationId) throws SQLEx
         }
         return String.join(",", grpIdList);
     }
+    public Integer getSubscriptionCountByApplicationId(Application application, String organization)
+            throws APIManagementException {
+
+        int subscriptionCount = 0;
+
+        try (Connection connection = APIMgtDBUtil.getConnection()) {
+            return getSubscriptionCountByApplicationId(connection, application, organization);
+        } catch (SQLException e) {
+            handleException("Failed to get SubscribedAPI of : application " + application.getName() + " in " +
```

Comment:
Let's use exception codes for error handling. https://github.com/wso2/apk/blob/main/common-java-libs/org.wso2.apk.apimgt.api/src/main/java/org/wso2/apk/apimgt/api/ExceptionCodes.java

Reply:
resolved